inc/admin/dashboard/class-networkdashboard.php

```diff
@@ -55,4 +63,140 @@ protected function getTotalNumberOfBooks(): int {
+			],
+			[
+				'plugin'      => 'pressbooks-cas-sso/pressbooks-cas-sso.php',
+				'link'        => network_admin_url( 'admin.php?page=pb_cas_admin' ),
+			],
+		];
+
+		// Check for active SSO plugin and add configuration
+		foreach ( $sso_configurations as $sso_configuration ) {
+			if ( is_plugin_active( $sso_configuration['plugin'] ) ) {
+				$sso_item = [
+					'id'          => 'network_checklist_configure_sso',
+					'title'       => __( 'Configure Single Sign On (SSO)', 'pressbooks' ),
+					'link'        => $sso_configuration['link'],
+					'description' => __( 'Allow users to login with their existing institutional credentials' ),
+					'checked'     => get_network_option( null, 'network_checklist_configure_sso', false ),
+				];
+
+				array_splice( $items, 3, 0, [ $sso_item ] );
+				break;
+			}
+		}
+
+		// Check if Network Analytics plugin is activated
+		if ( is_plugin_active( 'pressbooks-network-analytics/pressbooks-network-analytics.php' ) ) {
+			$items[] = [
+				'id' => 'network_checklist_join_forum',
+				'title' => __( 'Join the Pressbooks Community Forum', 'pressbooks' ),
+				'link' => 'https://pressbooks.community/invites/Rqa9J1wYUN',
+				'description' => __( 'Chat with other network managers in a dedicated group' ),
+				'checked' => get_network_option( null, 'network_checklist_join_forum', false ),
+			];
+			$items[] = [
+				'id' => 'network_checklist_book_meeting',
+				'title' => __( 'Complete your onboarding', 'pressbooks' ),
+				'link' => env( 'PB_CHECKLIST_BOOKING_URL' ),
+				'description' => __( 'Book a meeting to discuss any remaining questions with us' ),
+				'checked' => get_network_option( null, 'network_checklist_book_meeting', false ),
+			];
+		}
+
+		return $items;
+	}
+
+	public static function storeCheck(): void {
+		check_ajax_referer( 'pb-dashboard-checklist' );
+		$item = sanitize_text_field( wp_unslash( $_POST['item'] ?? '' ) );
+		$current = get_network_option( null, $item );
+		$updated = update_network_option( null, $item, ! $current );
+		if ( $updated ) {
+			wp_send_json_success(
+				[
+					'checked' => $updated,
+					'completed' => ( new self() )->checkIfAllChecked(),
+				]
+			);
+		}
+	}
+
+	public function checkIfAllChecked(): bool {
+		$checklist = $this->getNetworkChecklist();
+		foreach ( $checklist as $item ) {
+			if ( ! $item['checked'] ) {
+				return false;
+			}
+		}
+		return true;
+	}
+
+	/**
+	 * Check if the checklist should be displayed if the network is older than 3 months
+	 *
+	 * @return bool
+	 */
+	public function shouldDisplayChecklist(): bool {
+		global $wpdb;
+
+		// Get the root site creation date
+		$network_creation_date = $wpdb->get_var( $wpdb->prepare( "SELECT registered FROM $wpdb->blogs WHERE site_id = %s", get_main_network_id() ) );
+
+		if ( $network_creation_date ) {
+			$current_date = current_time( 'Y-m-d' ); // Use a non-timestamp format
+			$months_ago = strtotime( env( 'PB_CHECKLIST_NETWORK_CREATION_MONTHS_AGO' ) );
```

Comment:
We should add a check here to ensure the env variable is defined, especially for open-source users.
Right now if the variable is not defined I got:
```Deprecated: strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in /app/web/app/plugins/pressbooks/inc/admin/dashboard/class-networkdashboard.php on line 195 ```

Maybe adding this at the beginning?
```php
if ( ! env( 'PB_CHECKLIST_NETWORK_CREATION_MONTHS_AGO' )  ) {
    return false;
}
```